pom.xml

```diff
@@ -30,6 +28,40 @@
         <powermock.version>2.0.2</powermock.version>
         <httpClient.version>4.5.1</httpClient.version>
     </properties>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <version>0.7.5.201505241946</version>
```

Comment:
In other repos we used version=0.8.5, is it ok to use this?

Reply:
As of now i didnt see any issue